polybar-scripts/notification-spacex/notification-spacex.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
 
-spacex_launch=$(curl -sf https://api.spacexdata.com/v5/launches/next)
+spacex_launch=$(curl -sf "https://ll.thespacedevs.com/2.2.0/launch/upcoming/?lsp__name=SpaceX&status__ids=1&limit=1")
 
 if [ -n "$spacex_launch" ]; then
-    spacex_precision=$(echo "$spacex_launch" | jq -r '.date_precision' )
-    spacex_timestamp=$(echo "$spacex_launch" | jq -r '.date_unix' )
+    spacex_precision=$(printf "%s" "$spacex_launch" | jq -r '.results[0].net_precision.abbrev' )
```

Comment:
Is there a reason for printf here?
Just to make it consistent echo would be great, even if I know that printf ist maybe faster.